doc/nrf/libraries/modem/location.rst

```diff
@@ -40,6 +40,7 @@ Each location method has its own implementation for the location retrieval:
      precedence as the transport method of GNSS assistance data.
    * Note that acquiring GNSS fix only starts when LTE connection, more specifically Radio Resource Control (RRC) connection, is idle.
      Also, if A-GPS is not used and Power Saving Mode (PSM) is enabled, Location library will wait for the modem to enter PSM.
+   * Selectable location accuracy (low/normal/high).
```

Comment:
I would like to have a more verbose description of different accuracy modes somewhere in a single place. It does not need to be long but just mention the tricks that are used to achieve low and high accuracies (since they don't have any standardized meaning in the GNSS context)

Reply:
location.h described how high accuracy works but user needs to dig into modem lib documentation to understand low accuracy

Reply:
What about the Location library API? `location_gnss_config` already describes high accuracy, we could add description for low accuracy there as well.

Reply:
Updated the API, now both low and high accuracy are described.